src/nv_ingest/extraction_workflows/pdf/pdfium_helper.py

```diff
@@ -449,3 +475,24 @@ def pdfium_extractor(
         extracted_data.append(doc_text_meta)
 
     return extracted_data
+
+
+def _get_yolox_model_name(config, default_model_name="nv-yolox-page-elements-v1"):
```

Comment:
I don't think this should live in pdfium_helper. Maybe yolox.py, or something where its available to anyone else using it.

Reply:
Moved to yolox.py.